documentation/course/e2/README.md

```diff
@@ -1798,6 +1932,21 @@ TODO @tchojnacki: Dodać diagram pakietów, opis architektury.
 
 TODO @jakubzehner: Dodać diagram pakietów, opis architektury i endpointy.
 
+### API
+
+| **Rola**    | **Metoda** | **Endpoint**                   | **Wymagania** | **Opis**                                                                                                                |
```

Comment:
The public API seems to have all the necessary endpoints, I've also validated it with a source which shall not be named here and all seems good.